components/chat/ToolCallApproval.tsx

```
@@ -0,0 +1,101 @@
+  onApprove: () => void;
+  onReject: () => void;
+}
+
+export function ToolCallApproval({
+  toolCall,
+  isOpen,
+  onApprove,
+  onReject,
+}: ToolCallApprovalProps) {
+  // Format the tool call arguments for display
+  const formatArgs = (args: Record<string, any>) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)